frontend/packages/console-shared/src/components/formik-fields/SyncedEditorField.tsx

```diff
@@ -120,7 +155,15 @@ const SyncedEditorField: React.FC<SyncedEditorFieldProps> = ({
     setDisabledFormAlert(formContext.isDisabled);
   }, [formContext.isDisabled]);
 
-  return (
+  React.useEffect(() => {
```

Comment:
Why do you need this effect?

Reply:
we need this to update the last viewed user setting when the component loads.
this becomes useful when the last viewed type is different from the preferred type.